Sources/swift-container-plugin/Documentation.docc/build.md

```diff
@@ -0,0 +1,42 @@
+# Build and package your service
+
+## Overview
+
+`build-container-image` is a command plugin which takes care of building your service, packaging it in a container image and uploading it to a container registry, all in one command:
+
+```shell
+% swift package --swift-sdk x86_64-swift-linux-musl \
+        build-container-image --from swift:slim --repository registry.example.com/myservice
+```
+
+* The `--swift-sdk` argument specifies the Swift SDK with which to build the executable.   In this case we are using the Static Linux SDK, which was installed earlier, to build an statically-linked x86_64 Linux binary
```

Comment:
```suggestion
* The `--swift-sdk` argument specifies the Swift SDK with which to build the executable.   In this case we are using the Static Linux SDK, installed earlier, to build an statically-linked x86_64 Linux binary.
```

Reply:
Good idea.  Might also help to make `installed earlier` a link back to the installation section.

Reply:
Good idea.  Might also help to make `installed earlier` a link back to the installation section.